rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -18,7 +18,9 @@ include::modules/rosa-update-cli-tool.adoc[]
 [id="rosa-q2-2023_{context}"]
 === Q2 2023
 
-* {product-title} (ROSA) is now available in the United Arab Emirates (`me-central-1`) region.
+* **ROSA CLI update:** The ROSA CLI (`rosa`) has been updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.23[release notes]. For more information about the ROSA CLI (`rosa`,) see xref:../rosa_cli/rosa-get-started-cli.html[Getting started with the rosa CLI].
+
+* **ROSA regional availability update:** {product-title} (ROSA) is now available in the United Arab Emirates (`me-central-1`) region.
```

Comment:
The ROSA regional availability section is the only one that has the "For more information..." as a separate paragraph. Can you change this to be on the same line as the rest of the bullet, or make the other ones separate paragraphs too, please?

Reply:
Fixed!